LibOS/shim/src/bookkeep/shim_vma.c

```diff
@@ -353,8 +369,8 @@ static inline void __drop_vma (struct shim_vma * vma)
 static inline void
 __assert_vma_flags (const struct shim_vma * vma, int flags)
 {
-    if (!(vma->flags & VMA_UNMAPPED)
-          && (vma->flags & VMA_INTERNAL) != (flags & VMA_INTERNAL)) {
+    if (!(vma->flags & VMA_UNMAPPED) &&
+        VMA_TYPE(vma->flags) != VMA_TYPE(flags)) {
```

Comment:
Please fix indentation here, it looks now like the second like of the `if` condition belonged to the code block.

Reply:
Done.